src/Footer/Footer.js

```diff
@@ -1,4 +1,4 @@
 import './Footer.css';
 export default function Footer() {
-  return <footer></footer>;
+  return <footer>nikihite3@gmail.com</footer>;
```

Comment:
The contact email here needs to be passed in as a prop to your Footer component, instead of being hardcoded into it